docs/shared-cluster.md

```diff
@@ -0,0 +1,68 @@
+# Shared Cluster
+
+The shared cluster now lives in the AME tenant. We have access to credentials to use the cluster, but any "operational" action must go through JIT and the Azure Portal. 
+
+## Overview
+
+The following diagram is the overview of where our shared cluster lives, and how we access it.
+
+* Here is a link to the living lucid chart diagram: [here](https://lucid.app/lucidchart/1e415fe2-af56-4409-abc6-3bdf96f1bffd/edit?beaconFlowId=AB8BF83B17AD4D23&invitationId=inv_a6e62e97-2bcf-4c3a-b7b0-5dada5ea075d&page=0_0#)
+
+## Diagram
+
+![alt text](img/sharedcluster.png "Shared Cluster Overview")
+
+
+
+## Authentication
+
+We have the kubeadmin credentials as well as the kubeadmin kubeconfig file. You can use either to authenticate to the cluster.
+
+* Make secrets:
+
+```
+SECRET_SA_ACCOUNT_NAME=rharosecretsdev make secrets
+```
+
+* `oc login`, assuming `env` sources `secrets/env`:
+```
+. ./env
+make shared-cluster-login
+```
+
+* Use kubeconfig
+```
+export KUBECONFIG=$(pwd)/secrets/shared-cluster.kubeconfig
```

Comment:
What about using 
```bash
$PWD 
```
instead of 
```bash
$(pwd)
``` 
?
It appears more often in our codebase and it doesn't require the execution of a command in a sub-shell.

Reply:
done